Extension/src/LanguageServer/client.ts

```diff
@@ -3439,10 +3449,55 @@ export class DefaultClient implements Client {
             }
         }
 
-        if (uri === undefined || range === undefined) {
+        if (uri === undefined || range === undefined || editor === undefined) {
             return;
         }
 
+        let formatParams;
```

Comment:
I'm not sure why lint didn't flag this.  Variables should have types associated with them.  Could you change this to `let formatParams: FormatParams | undefined`?

Reply:
@Colengms A while back there was a change to the linter policy to allow variables without a type.

Reply:
...but I agree that it's better to set the type unless it's obvious from the code, i.e. treat it like you would "auto" in C++.